src/components/Book.jsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
+import axios from 'axios';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
 
 const Book = (props) => {
   const { id, title, author } = props;
   const dispatch = useDispatch();
   const callRemoveBook = (id) => {
-    dispatch(removeBook((id)));
+    const deleteUrl = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/OoMg7JW7xItdmWuHqq1t/books/${id}`;
+    axios.delete(deleteUrl, {
+      item_id: id,
+    }).then(() => {
+      dispatch(removeBook((id)));
```

Comment:
- **[OPTIONAL]** I think it would be better to put this functionality in the books redux file